cmd/monaco/runner/runner.go

```diff
@@ -37,7 +37,6 @@ func Run() int {
 	rootCmd := BuildCli(afero.NewOsFs())
 
 	if err := rootCmd.Execute(); err != nil {
```

Comment:
how does that impact the logs? IIRC logging how execution failed as the very last line of log was added intentionally